xonsh/ptk_shell/shell.py

```diff
@@ -191,10 +148,14 @@ def __init__(self, **kwargs):
         self.push = self._push
 
         ptk_args.setdefault("history", self.history)
-        if not XSH.env.get("XONSH_COPY_ON_DELETE", False):
-            disable_copy_on_deletion()
-        if HAVE_SYS_CLIPBOARD:
-            ptk_args.setdefault("clipboard", PyperclipClipboard())
+        if HAVE_SYS_CLIPBOARD and (
+            XSH.env.get("XONSH_USE_SYSTEM_CLIPBOARD", False)
+            or XSH.env.get("XONSH_COPY_ON_DELETE", False)
```

Comment:
Here the copy_on_delete controls all copy actions. It should only override the on_delete functions

Reply:
My understanding is that outside of VI mode, "copy on delete" is the only possible clipboard interaction anyway.  
So I was thinking that just deprecating the `XONSH_COPY_ON_DELETE` and making it equivalent to `XONS_USE_SYSTEM_CLIPBOARD` would be good enough since there would be no behavior change if `$VI_MODE == False`, and no-one using VI mode would have used this setting anyway (at least I can't think of a use-case for using the system clipboard for `p` but the in-memory clipboard for `dd`?). Hence #5155.

Of course I could leave the old implementation be, but it will be messier so I want to be sure that there is a real use-case we'd be supporting.

Reply:
With PTK, we can select and copy the text using shift and ctrl+c like text editors. This would be disabling that case. Also ctrl+x is also implemented in PTK